extensions/wikia/GlobalFooter/GlobalFooterController.class.php

```diff
@@ -49,6 +47,7 @@ private function getGlobalFooterLinks() {
 				"Global Footer's links not found in messages",
 				[ 'exception' => new Exception() ]
 			);
+			wfProfileOut( __METHOD__ );
```

Comment:
Oh, you're right. I noticed a return without wfProfileOut, but it seems there was one even before WikiaLogger.